### ISSUE TITLE
Time picker spinner is not working in release mode, but working fine in debug

### ISSUE THREAD

**Issue by kashifg4171** [icemanbsi/flutter_time_picker_spinner#7]
![WhatsApp Image 2020-06-03 at 5 30 18 PM](https://user-images.githubusercontent.com/47936582/83636843-13a36280-a5c0-11ea-9320-f50f850026aa.jpeg)

**Comment by defemz** [comment]
I am currently experiencing the same issue. really frustrating

**Comment by kashifg4171** [comment]
> I am currently experiencing the same issue. really frustrating

Did you find any solution?

**Comment by defemz** [comment]
> > I am currently experiencing the same issue. really frustrating
> 
> Did you find any solution?

No. I have not. Seems the authors of the package is not responding timely to issues, I am thinking of looking for alternative Time picker.

**Comment by OmarYehiaDev** [comment]
> > > I am currently experiencing the same issue. really frustrating
> > 
> > 
> > Did you find any solution?
> 
> No. I have not. Seems the authors of the package is not responding timely to issues, I am thinking of looking for alternative Time picker.

@defemz Did u found an alternative? I'm stuck in the same situation :"(
